File: Serene/Serene.Script/Meeting/MeetingDecision/MeetingDecisionDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingDecisionId"), NameProperty("Description"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingDecision"), LocalTextPrefix("Meeting.MeetingDecision"), Service("Meeting/MeetingDecision")]
    public class MeetingDecisionDialog : EntityDialog<MeetingDecisionRow>
    {
    }
}

File: Serene/Serene.AspNetCore/Modules/Meeting/MeetingDecision/MeetingDecisionDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingDecisionDialog extends Serenity.EntityDialog<MeetingDecisionRow, any> {
        protected getFormKey() { return MeetingDecisionForm.formKey; }
        protected getIdProperty() { return MeetingDecisionRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingDecisionRow.localTextPrefix; }
        protected getNameProperty() { return MeetingDecisionRow.nameProperty; }
        protected getService() { return MeetingDecisionService.baseUrl; }

        protected form = new MeetingDecisionForm(this.idPrefix);

    }
}